Makefile

```diff
@@ -10,6 +12,16 @@ build: $(go_sources)
 	mkdir -p build
 	CGO_ENABLED=0 go build -o build/foreman_worker $^
 
+.PHONY: data
+data: build/data/com.redhat.Yggdrasil1.Worker1.foreman.conf build/data/com.redhat.Yggdrasil1.Worker1.foreman.service
+
+.PHONY: install
+install: build data
+	install -D -m 755 build/foreman_worker $(DESTDIR)/$(LIBEXECDIR)/foreman_worker
+	install -D -m 644 build/data/com.redhat.Yggdrasil1.Worker1.foreman.conf $(DESTDIR)/usr/share/dbus-1/system.d/com.redhat.Yggdrasil1.Worker1.foreman.conf
+	install -D -m 644 data/dbus_com.redhat.Yggdrasil1.Worker1.foreman.service $(DESTDIR)/usr/share/dbus-1/system-services/com.redhat.Yggdrasil1.Worker1.foreman.service
+	install -D -m 644 build/data/com.redhat.Yggdrasil1.Worker1.foreman.service $(DESTDIR)/usr/lib/systemd/system/com.redhat.Yggdrasil1.Worker1.foreman.service
```

Comment:
Huh, interesting. I would expect the rpm build to fail due to additional files. But the build seems to pass and those files are note present in the resulting rpm.